3p/integration.js

```diff
@@ -1,4 +1,4 @@
-/**
+﻿/**
```

Comment:
I'm not sure why it the change was applied by git, but there is definitely something that went wrong with the PR, as other changes appear when they should not. (e.g. the objects named `git` and `cd` with no changes).

Please remove those two empty files. 

As for the first line change, that's probably fine after squash, so don't worry about it.

**Important:** Also note that the Google CLAs need to be signed by all parties in this PR. I won't be able to merget these changes unless @taisuke-oomi signs the agreement, as they made commits as part of this PR.